automated install/basic-install.sh

```diff
@@ -968,7 +979,7 @@ fi
 
 echo ":::"
 echo "::: The install log is located at: /etc/pihole/install.log"
-echo "::: View the web interface at http://pi.hole/admin or http://${IPv4addr%/*}/admin"
+echo "::: View the web interface at http://pi.hole/admin or http://${IPv4_address%/*}/admin"
 }
 
 main "$@"
```

Comment:
when dry run is set, do not run the main / anything. `if [[ -z "$DRYRUN" ]] ; main "$@ ; fi;` so it would always run main by default.

then if I wanted to write a test for one function I could do something like:
```
DRYRUN=TRUE . automated install/basic-install.sh;
useIPv6dialog
assert $IPv6addr == 1.2.3.4
```